.tfdev

```diff
@@ -1,12 +1,8 @@
-version_info {
-  commit_var  =    "main.GitCommit"
-  version_var =    "github.com/hashicorp/terraform/version.Version"
-  prerelease_var = "github.com/hashicorp/terraform/version.Prerelease"
-}
+version_info {	
+  commit_var  =    "main.GitCommit"	
+  version_var =    "github.com/hashicorp/terraform/version.Version"	
+  prerelease_var = "github.com/hashicorp/terraform/version.Prerelease"	
+}	
```

Comment:
There seems to be some extra whitespace lurking at the end of these lines, for some reason... :thinking: